augur/tasks/git/facade_tasks.py

```diff
@@ -457,43 +496,13 @@ def facade_phase(repo_git):
         
         # Figure out what we need to do
         limited_run = session.limited_run
-        delete_marked_repos = session.delete_marked_repos
```

Comment:
I think we are getting rid of many of the confusing different settings. yes?

Reply:
Yes some have been removed but not all. Delete_marked_repos particularly should probably be removed because we never remove repos.